### DISCUSSION TITLE
Problem in running LIS DA using SMOPS ASCAT data,

### DISCUSSION THREAD

**Discussion by srikar-iist** [NASA-LIS/LISF#1408] [Q&A]
Hi, Im trying to run DA using SMOPS-ASCAT data but i am getting the following error.

[lis_da]$ ./LIS -f lis.config_noah36_smapda
****************Error****************************
setup routine for DA obs SMOPS-ASCAT soil moisture is not defined
program will seg fault.....
****************Error****************************
forrtl: severe (174): SIGSEGV, segmentation fault occurred
Image              PC                Routine            Line        Source             
LIS                0000000000FC0C9A  Unknown               Unknown  Unknown
libpthread-2.17.s  00002B06A6CAD5D0  Unknown               Unknown  Unknown
LIS                00000000004DA83D  Unknown               Unknown  Unknown
LIS                00000000004B5BAD  lis_daobservation         249  LIS_DAobservationsMod.F90
LIS                0000000000DC67FC  retrospective_run          91  retrospective_runMod.F90
LIS                0000000000AE0EC6  MAIN__                     53  lisdrv.F90
LIS                00000000004141D2  Unknown               Unknown  Unknown
libc-2.17.so       00002B06A71DE3D5  __libc_start_main     Unknown  Unknown
LIS                00000000004140E9  Unknown               Unknown  Unknown

I am attaching my lis.config file also - my LIS version is LIS 7.2

Kindly help.
[lis.config_noah36_smapda.txt](https://github.com/NASA-LIS/LISF/files/12557506/lis.config_noah36_smapda.txt)

**Comment by dmocko** [comment]
Hi @srikar-iist 

I don't believe that SMOPS-ASCAT DA was availabe in LIS version 7.2

Please use the latest public version - 7.4.3 - and try again.  If you continue to have an error, please open a new Discussion.
